docs/CREATE4646.md

```diff
@@ -0,0 +1,21 @@
+# 4626 Market Creation
+
+In order to create a market, you must first have a market creator account. This account will be funded with an initial mint amount to establish the share price of the market.
+
+## Deployment Steps
+
+1. Set REWARDS_RECEIVER to the address you want to receive rewards in Addresses.sol on the network you're deploying on.
+2. Fund the deployer with the following token amounts:
```

Comment:
~~What's the reason for deployer to need tokens? Is it to test deposits?~~

Nvm, found it below.